playbooks/roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -0,0 +1,43 @@
+filebeat:
+  # List of prospectors to fetch data.
```

Comment:
Same here, I think we can do without this, as the config format is officially documented elsewhere.